index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Please consider adding the `lang` attribute to the `html` tag. The `lang` attribute is used to identify the language of a webpage and is used by search engines to return language-specific results in search results. If the webpage is in english, you could to this
```HTML
<html lang="en">
```

Reply:
[OPTIONAL]
Great asset to point out as the SEO does really matter for the UX. :smile: 
It would be also great to give the hint of generating the `HTML boiler plate` as It provides even more crucial initial points to start with .